modules/oc-mirror-workflows-delete-v2.adoc

```diff
@@ -48,3 +48,10 @@ Consider using the mirror-to-disk and disk-to-mirror workflows to reduce mirrori
 In the image delete workflow, oc-mirror plugin v2 deletes only the manifests of the images, which does not reduce the storage occupied in the registry.
 
 To free up storage space from unnecessary images, such as those with deleted manifests, you must enable the garbage collector on your container registry. With the garbage collector enabled, the registry will delete the image blobs that no longer have references to any manifests, thereby reducing the storage previously occupied by the deleted blobs. Enabling the garbage collector differs depending on your container registry. 
+
+[IMPORTANT]
+====
+To skip deleting the entire operator catalog when using the delete feature, you must list the specific operators under the operator catalog in the `DeleteImageSetConfiguration`. This ensures that only the specified operators are deleted, not the entire catalog image.
```

Comment:
@subhtk Do you think it is worth adding an example so that it will be useful for the customer ? 

@sherine-k @aguidirh @lmzuccarelli WDYT ?

Reply:
We can add an example.

Reply:
I agree with @kasturinarra , also LGTM

Reply:
I think you should add `entire operator catalog image`

Reply:
@lmzuccarelli Have added the examples. Please have a look.